projects/plugins/social/src/class-note.php

```diff
@@ -11,14 +11,44 @@
  * Register the Jetpack Social Note custom post type.
  */
 class Note {
+	const JETPACK_SOCIAL_NOTE_CPT     = 'jetpack-social-note';
+	const FLUSH_REWRITE_RULES_FLUSHED = 'jetpack_socia_rewrite_rules_flushed';
```

Comment:
There's a minor typo here (not that it matters too much)
```suggestion
	const FLUSH_REWRITE_RULES_FLUSHED = 'jetpack_social_rewrite_rules_flushed';
```